src/ontology/Modules/occo_terms.owl

```diff
@@ -464,6 +513,249 @@
+        <obo:IAO_0000117>Robin McGill</obo:IAO_0000117>
+        <obo:IAO_0000117>Sam Smith</obo:IAO_0000117>
+        <rdfs:label>academic degree</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OCCO_00000126 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OCCO_00000126">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OCCO_00000118"/>
+        <obo:IAO_0000115>A credential awarded by a government or government-authorized agency designed to signify legal authority to engage in specified actions without being liable to sanctions or penalties that would otherwise be associated with those actions.</obo:IAO_0000115>
+        <obo:IAO_0000117>Bill Duncan</obo:IAO_0000117>
+        <obo:IAO_0000117>Bill Hogan</obo:IAO_0000117>
+        <obo:IAO_0000117>Finn Wilson</obo:IAO_0000117>
+        <obo:IAO_0000117>Giacomo De Colle</obo:IAO_0000117>
+        <obo:IAO_0000117>Jie Zheng</obo:IAO_0000117>
+        <obo:IAO_0000117>John Beverley</obo:IAO_0000117>
+        <obo:IAO_0000117>Matthew Diller</obo:IAO_0000117>
+        <obo:IAO_0000117>Oliver He</obo:IAO_0000117>
+        <obo:IAO_0000117>Robin McGill</obo:IAO_0000117>
+        <obo:IAO_0000117>Sam Smith</obo:IAO_0000117>
+        <rdfs:label>license</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OCCO_00000127 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OCCO_00000127">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/BFO_0000023"/>
+        <obo:IAO_0000115>A role borne by an organization that has been accredited by a quality assurance group through a deontic declaration having action regulation output which authorizes the organization to bestow credentials according to quality control standards established or enforced by the accrediting group.</obo:IAO_0000115>
+        <obo:IAO_0000117>Bill Duncan</obo:IAO_0000117>
+        <obo:IAO_0000117>Bill Hogan</obo:IAO_0000117>
+        <obo:IAO_0000117>Finn Wilson</obo:IAO_0000117>
+        <obo:IAO_0000117>Giacomo De Colle</obo:IAO_0000117>
+        <obo:IAO_0000117>Jie Zheng</obo:IAO_0000117>
+        <obo:IAO_0000117>John Beverley</obo:IAO_0000117>
+        <obo:IAO_0000117>Matthew Diller</obo:IAO_0000117>
+        <obo:IAO_0000117>Oliver He</obo:IAO_0000117>
+        <obo:IAO_0000117>Robin McGill</obo:IAO_0000117>
+        <obo:IAO_0000117>Sam Smith</obo:IAO_0000117>
+        <rdfs:label>credential grantor role</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OCCO_00000128 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OCCO_00000128">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/BFO_0000015"/>
+        <obo:IAO_0000115>A process in which a credential granting agency confirms that an agent has satisfied the requirements for being awarded a credential, having output a credential that is about the agent.</obo:IAO_0000115>
```

Comment:
Definition reads "A social act in which a credential granting agency asserts that an organism has satisfied the
requirements for being awarded a credential, having as output a credential about the organism.

Reply:
I updated the definitions based on the OccO paper that Finn shared with me. 

The definition of 'occupational credential holder' is 'An organism that an occupation credential is about.'. I did not change its parent from 'Homo sapiens' to 'organism' since we have not imported 'organism' in OccO yet. 
The OBI ['organism'](http://purl.obolibrary.org/obo/OBI_0100026) is equal to 'Viruses' or 'Bacteria' or 'Archaea or Eukaryota'. The CARO ['organism'](http://purl.obolibrary.org/obo/CARO_0001010) is 'Material anatomical entity that is a member of an individual species or is a viral or viroid particle'. Please let me know which 'organism' OccO thought be a 'occupational credential holder'. Then I will update it in the OccO. Thanks!